doc/api/buffer.md

```diff
@@ -846,11 +846,11 @@ _may contain sensitive data_. Use [`buf.fill(0)`][`buf.fill()`] to initialize
 such `Buffer` instances with zeroes.
 
 When using [`Buffer.allocUnsafe()`][] to allocate new `Buffer` instances,
-allocations under 4 KiB are sliced from a single pre-allocated `Buffer`. This
-allows applications to avoid the garbage collection overhead of creating many
-individually allocated `Buffer` instances. This approach improves both
-performance and memory usage by eliminating the need to track and clean up as
-many individual `ArrayBuffer` objects.
+allocations less than `Buffer.poolSize >>> 1` (4 KiB by default) are sliced
```

Comment:
The buffer poolSize was changed to `8KiB` 9 years ago.

Reply:
```suggestion
allocations less than `Buffer.poolSize >>> 1` (8 KiB by default) are sliced
```

Reply:
Thanks for looking it over! 😊

Reply:
Looking at this again, I think 4KiB is referring to half the pool size, not the default pool size itself. Does it read ambiguously?

Reply:
The last suggestion I think it will be better, 4KiB when default pool size.

Reply:
Agreed, just made the update